src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -101,6 +104,31 @@ private void CheckFactoryWasUnregisteredFromDataSource(SnapshotFactory factory,
             mockSonarErrorDataSource.Verify(x => x.RemoveFactory(factory), times);
         }
 
+        [TestMethod]
+        public void OnBufferChange_ErrorListIsUpdated()
```

Comment:
Previously there weren't any tests for the buffer change handling.
This at least tests that an analysis is not triggered and the error list is updated.

We can't easily check whether the taggers are called since to do that we need some valid spans.